### ISSUE TITLE
Error when enabling seasons on new world

### ISSUE THREAD

**Issue by Paradox101** [casperge/RealisticSeasons-issues#93]
Getting this error when I use /rs set winter

```
Console23:45:38
org.bukkit.command.CommandException: Unhandled exception executing command 'rs' in plugin RealisticSeasons v7.8
at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
at org.bukkit.craftbukkit.v1_18_R1.CraftServer.dispatchCommand(CraftServer.java:897) ~[paper-1.18.1.jar:git-Paper-105]
at net.minecraft.server.network.ServerGamePacketListenerImpl.handleCommand(ServerGamePacketListenerImpl.java:2268) ~[?:?]
at net.minecraft.server.network.ServerGamePacketListenerImpl.handleChat(ServerGamePacketListenerImpl.java:2079) ~[?:?]
at net.minecraft.server.network.ServerGamePacketListenerImpl.handleChat(ServerGamePacketListenerImpl.java:2060) ~[?:?]
at net.minecraft.network.protocol.game.ServerboundChatPacket.handle(ServerboundChatPacket.java:46) ~[?:?]
at net.minecraft.network.protocol.game.ServerboundChatPacket.a(ServerboundChatPacket.java:6) ~[?:?]
at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$1(PacketUtils.java:56) ~[?:?]
at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.18.1.jar:git-Paper-105]
at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:149) ~[?:?]
at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:23) ~[?:?]
at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1413) ~[paper-1.18.1.jar:git-Paper-105]
at net.minecraft.server.MinecraftServer.c(MinecraftServer.java:189) ~[paper-1.18.1.jar:git-Paper-105]
at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:122) ~[?:?]
at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1391) ~[paper-1.18.1.jar:git-Paper-105]
at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1384) ~[paper-1.18.1.jar:git-Paper-105]
at net.minecraft.util.thread.BlockableEventLoop.runAllTasks(BlockableEventLoop.java:110) ~[?:?]
at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1518) ~[paper-1.18.1.jar:git-Paper-105]
at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1264) ~[paper-1.18.1.jar:git-Paper-105]
at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317) ~[paper-1.18.1.jar:git-Paper-105]
at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.NullPointerException: Cannot invoke "me.casperge.realisticseasons.temperature.TemperatureSettings.getMinTemperature(me.casperge.realisticseasons.season.Season)" because "this.tempsettings" is null
at me.casperge.realisticseasons.temperature.TempUtils.generateNewBaseTemperature(TempUtils.java:281) ~[RealisticSeasons-7.8.jar:?]
at me.casperge.realisticseasons.temperature.TemperatureManager.loadWorld(TemperatureManager.java:76) ~[RealisticSeasons-7.8.jar:?]
at me.casperge.realisticseasons.season.SeasonManager.setSeason(SeasonManager.java:271) ~[RealisticSeasons-7.8.jar:?]
at me.casperge.realisticseasons.season.SeasonManager.setSeason(SeasonManager.java:286) ~[RealisticSeasons-7.8.jar:?]
at me.casperge.realisticseasons.commands.RealisticSeasonsCommand.setSeason(RealisticSeasonsCommand.java:307) ~[RealisticSeasons-7.8.jar:?]
at me.casperge.realisticseasons.commands.RealisticSeasonsCommand.setByPlayer(RealisticSeasonsCommand.java:356) ~[RealisticSeasons-7.8.jar:?]
at me.casperge.realisticseasons.commands.RealisticSeasonsCommand.onCommand(RealisticSeasonsCommand.java:77) ~[RealisticSeasons-7.8.jar:?]
at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
... 21 more
```

My config:

` Please note that spawn-chance-animals, flower-chance-in-spring and entities-pumpkin-chance have a 1/x chance of happening. 
 This means that increasing the chance will reduce the amount spawned
 spawn-chance-animals is a 1/x chance of spawning an animal each chunk
 flower-chance-in-spring is a 1/x chance per block
modify-blocks: true
modify-daynight-length: true
modify-weather-cycle: true
modify-biome-on-client: true
dynmap-pause-renders-in-winter: true
affect-crop-growth: true
delay-per-operation: 4
automatic-season-cycling: true
send-message-on-season-cycle: true
spawn-ice-in-winter: true
disable-snow-placement-all-snowman: false
snowman-wear-pumpkin: false
snowman-attack-mobs: false
max-berrybushes-per-chunk: 20
disable-berry-damage-for-entities: true
flower-chance-in-spring: 20
regrow-grass-everywhere: false
spawn-mushrooms-in-fall: true
keep-natural-snow: true
snow-placement-enabled: true
entities:
  spawn-entities: true
  spawn-chance-animals: 10
  despawn-season-animals: true
  spring:
    spawn-extra-animals: true
    spawn-extra-bees: true
    spawn-extra-rabits: true
  summer:
    spawn-ocelots: true
    spawn-pandas: true
    spawn-parrots: true
    replace-zombies-with-husks: true
  fall:
    spawn-bats: true
    spawn-cave-spiders: true
    spawn-foxes: true
    spawn-mooshrooms: true
    spawn-extra-spiders: true
    give-entities-pumpkin: true
    entities-pumpkin-chance: 5
  winter:
    spawn-wolfs: true
    spawn-foxes: true
    spawn-polarbears: true
    spawn-snowman: true
    replace-skeletons-with-strays: true
`

**Comment by casperge** [comment]
Could you check for me if there are more errors at startup? I think this could be a result from another error.

**Comment by Paradox101** [comment]
No errors at all on startup. Seems to load just fine.

`[19:31:28] [Server thread/INFO]: [Multiverse-Portals] Loading Multiverse-Portals v4.2.1-b834
[19:31:28] [Server thread/INFO]: [GriefPrevention] Loading GriefPrevention v16.18-RC1-1-g06cc566
[19:31:28] [Server thread/INFO]: [RealisticSeasons] Loading RealisticSeasons v7.8
[19:31:28] [Server thread/INFO]: [Dynmap-GriefPrevention] Loading Dynmap-GriefPrevention v0.81
[19:31:28] [Server thread/INFO]: [RandomTeleport] Loading RandomTeleport v2.0.0.68
[19:31:28] [Server thread/INFO]: Server permissions file permissions.yml is empty, ignoring it`

**Comment by casperge** [comment]
Interesting, the error is coming from the temperature file. Did you disable the temperature system by setting "enabled" in the temperature.yml to false, or haven't you touched it at all?

**Comment by Paradox101** [comment]
Yes! its set to enabled: false

**Comment by casperge** [comment]
I think some part of my plugin doesn't like that, will fix it tomorrow. Will also comment here about progress. Thanks for your help!

**Comment by Paradox101** [comment]
Sure, thx for quick reply!

**Comment by casperge** [comment]
Fixed in v7.9, let me know if you come across any other issues
